public/css/app.css

```diff
@@ -2119,11 +2119,11 @@ select.form-control[multiple], select.form-control[size], textarea.form-control
     padding: .375rem .75rem;
     font-size: .9rem;
     line-height: 1.6;
-    border-radius: .25rem;
+    border-radius: .255rem;
     transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out
 }
 
-@media screen and (prefers-reduced-motion: reduce) {
+@media screen and (max-width: 768px) {
```

Comment:
You seem to have multiple media queries with the same condition. 

@media screen and (max-width: 768px).

You should combine the code of each into one query instead.

Reply:
this are code generated when I install the bootstrap. I just change the media queries to max-width tho.

Reply:
Got it. In that case, leave it as it is.